data/dbus/session.conf

```diff
@@ -0,0 +1,126 @@
+<!-- This configuration file controls Anaconda's special session message bus.
+     It is based on the /usr/share/dbus-1/session.conf file.
+
+     Author(s): Jiri Konecny <jkonecny@redhat.com
+-->
+
```

Comment:
Please, change the name of the file.

Reply:
The config file looks to be too strict, but that is fine for now.

Reply:
We can change the policy anytime we want.